scripts/backpressure_report/backpressure_report/lib/backpressure_window.py

```diff
@@ -0,0 +1,37 @@
+from datetime import timedelta
+
+
+def build_windows_from_events(backpressure_events, window_width_in_hours=1):
+    """
+    Generate histogram-friendly time windows with counts of backpressuring durations within each window.
```

Comment:
I think what what's in this PR is correct, but it's not what I would think of as a histogram. I imagine that would be something like binning the backpressure durations to get a sense of how long they are when they happen (and is there anything interesting in the duration distribution). Is there a different way I should be thinking about this?

Reply:
Right it's not a histogram just a barchart... 🤦